opensrp-anc/src/main/java/org/smartregister/anc/library/model/ContactVisit.java

```diff
@@ -213,7 +213,7 @@ private void processFormFieldKeyValues(String baseEntityId, JSONObject object, S
 
                     for (int i = 0; i < stepArray.length(); i++) {
                         JSONObject fieldObject = stepArray.getJSONObject(i);
-                        ANCFormUtils.processSpecialWidgets(fieldObject);
+                        org.smartregister.anc.library.util.ANCFormUtils.processSpecialWidgets(fieldObject);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.util.ANCFormUtils.processSpecialWidgets' due to existing import 'org.smartregister.anc.library.util.ANCFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)